ansible/gluster-cluster.yml

```diff
@@ -2,7 +2,7 @@
   hosts: storage
   roles:
     - role: stackhpc.gluster-cluster
```

Comment:
Are you running this role against Atomic?

Reply:
No, against openhpc cluster. I only run the mount_gluster role on Atomic. Will probably eventually need to adapt this for Atomic since it wouldn't work in its current state.